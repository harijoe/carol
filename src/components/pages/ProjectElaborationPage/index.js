import React from 'react'
import Helmet from 'react-helmet'

import { ProjectElaborationLayout } from 'components'
import { ProjectElaboration } from 'containers'

const ProjectElaborationPage = props => (
  <ProjectElaborationLayout {...props}>
    <Helmet>
      <meta name="robots" content="noindex, nofollow" />
    </Helmet>
    <ProjectElaboration />
  </ProjectElaborationLayout>
)

export default ProjectElaborationPage
