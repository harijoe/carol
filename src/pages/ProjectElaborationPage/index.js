import React from 'react'
import Helmet from 'react-helmet'

import ProjectElaborationLayout from 'layouts/ProjectElaborationLayout'
import ProjectElaboration from 'containers/ProjectElaboration'

const ProjectElaborationPage = props =>
  <ProjectElaborationLayout {...props}>
    <Helmet>
      <meta name="robots" content="noindex, nofollow" />
    </Helmet>
    <ProjectElaboration />
  </ProjectElaborationLayout>

export default ProjectElaborationPage
