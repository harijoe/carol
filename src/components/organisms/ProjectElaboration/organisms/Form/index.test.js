import React from 'react'
import { shallow } from 'enzyme'
import ProjectElaborationPage from './'

it('renders Form', () => {
  expect(shallow(<ProjectElaborationPage />)).toMatchSnapshot()
})
