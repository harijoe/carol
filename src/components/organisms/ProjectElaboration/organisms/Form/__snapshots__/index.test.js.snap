exports[`test renders Form 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Connect(Form(InjectIntl(Form)))
    destroyOnUnmount={true}
    enableReinitialize={false}
    forceUnregisterOnUnmount={false}
    form="ProjectElaboration"
    getFormState={[Function]}
    initialValues={undefined}
    keepDirtyOnReinitialize={false}
    persistentSubmitErrors={false}
    pure={true}
    shouldAsyncValidate={[Function]}
    shouldValidate={[Function]}
    touchOnBlur={true}
    touchOnChange={false} />,
  "nodes": Array [
    <Connect(Form(InjectIntl(Form)))
      destroyOnUnmount={true}
      enableReinitialize={false}
      forceUnregisterOnUnmount={false}
      form="ProjectElaboration"
      getFormState={[Function]}
      initialValues={undefined}
      keepDirtyOnReinitialize={false}
      persistentSubmitErrors={false}
      pure={true}
      shouldAsyncValidate={[Function]}
      shouldValidate={[Function]}
      touchOnBlur={true}
      touchOnChange={false} />,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <ReduxForm />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": ReduxForm {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Connect(Form(InjectIntl(Form)))
          destroyOnUnmount={true}
          enableReinitialize={false}
          forceUnregisterOnUnmount={false}
          form="ProjectElaboration"
          getFormState={[Function]}
          initialValues={undefined}
          keepDirtyOnReinitialize={false}
          persistentSubmitErrors={false}
          pure={true}
          shouldAsyncValidate={[Function]}
          shouldValidate={[Function]}
          touchOnBlur={true}
          touchOnChange={false} />,
        "_debugID": 2,
        "_renderedOutput": <Connect(Form(InjectIntl(Form)))
          destroyOnUnmount={true}
          enableReinitialize={false}
          forceUnregisterOnUnmount={false}
          form="ProjectElaboration"
          getFormState={[Function]}
          initialValues={undefined}
          keepDirtyOnReinitialize={false}
          persistentSubmitErrors={false}
          pure={true}
          shouldAsyncValidate={[Function]}
          shouldValidate={[Function]}
          touchOnBlur={true}
          touchOnChange={false} />,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ReduxForm />,
}
`;
